docs/posix-secondaries.adoc

```diff
@@ -0,0 +1,86 @@
+= Posix Secondaries aka IP Secondaries
+
+Secondary is a term introduced by link:https://uptane.github.io/uptane-standard/uptane-standard.html[the uptane spec].
+It refers to ECU software/firmware/package updates of which are managed by another ECU that hosts and runs Uptane compliant software. The later ECU is called Primary and the sotfware it runs in our case is called aktualizr.
+
+The goal of this doc is to guide a reader on configuring and running two executables that can be build out of the given repository source code and act as Primary and Secondary correspondingly.
+
+A key component of this repository is libaktualizr which is a library implementing Uptane protocol and providing customers and community with capability to build application(s) one of the purposes of which is to update software/firmware/packages in compliance with the Uptane standard. In addition to libaktualizr, an user can find in this repo another two components:
+
+. *aktualizr*, aka aktualizr primary - a reference implementation of an application utilizing libaktualizr, its binary/executable is called aktualizr. Aktualizr is used to emulate/"fake" Primary locally on an user's host;
+. *aktualizr-secondary*, aka posix or IP secondary - a reference implementation intended for running on Secondary ECU and capable to communicate with Primary ECU aktualizr via TCP/IP protocol (implemenation relies on Posix API). Aktualizr-secondary is used to "fake" Secondary ECU on an user's host.
```

Comment:
I would make it clear that "Aktualizr can be used to emulate a Primary locally...". The *can be* is critical, since it can also be used to update a real primary! Same goes for the secondary.

Reply:
agree